CarCostsTracker/CarCostsTracker/History/HistoryDisplayData.swift

```diff
@@ -13,3 +13,20 @@ import Viperit
 final class HistoryDisplayData: DisplayData {
     
 }
+
+struct HistoryCellData{
+    let costType: CostType
+    let costImage: UIImage
```

Comment:
Where is this used? Could it be moved to `CostType` enumeration?

Reply:
I used for display. and as CostType Enum implemented in HistoryCollectionViewCell